armi/reactor/tests/test_components.py

```diff
@@ -204,10 +204,6 @@ def test_fromComponent(self):
 class TestShapedComponent(TestGeneralComponents):
     """Abstract class for all shaped components"""
 
-    def test_getBoundingCircleOuterDiameter(self):
```

Comment:
Why are we removing this test?

Reply:
because we implemented it! but you're right, we should just update the test to actually test it, eh?